areas/js/02-reflection.js

```diff
@@ -11,10 +11,12 @@ function updateEmotion(msg) {
   if (!curEmotion || curEmotion.name !== msg.name) {
     curEmotion = msg;
     console.log('emotion has been updated to: ' + msg.name + ' (base: ' + msg.base + ', level: ' + msg.level +')');
+    showLoadingOverlay(curEmotion.name);
     updateInterface();
   }
 }
 
 function updateInterface() {
   $('#debug-info').text('CURRENT EMOTION: ' + curEmotion.name + ' (base: ' + curEmotion.base + ', level: ' + curEmotion.level +')')
+  setTimeout(hideLoadingOverlay, 2000);
```

Comment:
just thinking it could be helpful to pull some of this timing into a global variables file so it can be tweaked across zones as needed

Reply:
For sure! just pushed a little update that moves the timeout to the `showLoadingOverlay` function, along with a note about changing this in the future.